test/fixtures/access-control/common/models/account2.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "accountWithReplaceOnPUTfalse",
+  "plural": "accounts2",
```

Comment:
>Please use a more descriptive URL path than accounts2

Yes I could not come up with a good name :( Not sure if I understood your suggestion... are you suggesting to use?:

|  model name           | model plural  |
|-------------:| -----:|
| accountWithReplaceOnPUTfalse | accounts-updating |
| accountWithReplaceOnPUTtrue      |  accounts-replacing |

I applied name changes based on this table ^